lib/apollo/clients/base_client.rb

```diff
@@ -41,12 +57,13 @@ def handle_request_with_headers(url, http_verb, params: nil, body: nil)
       [body, result.headers]
     end
 
-    def handle_result(result)
-      if (result.status >= 200 && result.status < 300) || result.status == 404
+    def handle_results(result)
```

Comment:
Hey there is a bug here. The method is renamed looks like with an extra `s`

Reply:
My bad. Thanks for catching. Will fix it immediately.